css/style.css

```diff
@@ -0,0 +1,30 @@
+
+  /* box-shadow: 0px 5px 18px -5px rgba(0,0,0,0.5); */
+#logoText {
+  color: white;
+  width: 7em;
+  text-align: center;
+  font-size: 3em;
+  position: relative;
+  bottom: 4em;
+  left: 5em;
+  padding: .5em;
+  border: 1px solid white;
+  margin: 0;
+}
+
+
+body {
+  font-family: 'Loved by the King', cursive;
+  background-image: url('../img/wood.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+#salmon {
+  box-shadow: 20px 20px 18px -5px rgba(0,0,0,0.5);
+  padding: 10px;
+  background-image: url("../img/weathered.jpg");
+  background-size: 100%;
+  border: 15px solid rgb(70, 1, 1);
+}
```

Comment:
nitpick: this angry red symbol means you haven't left a blank line at the end of your file; in general, GitHub wants you to leave a blank line at the end of every file.